app/src/main/java/cs465/illinois/edu/dogonthequad/CreateMeetupLocation.java

```diff
@@ -0,0 +1,23 @@
+package cs465.illinois.edu.dogonthequad;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+
+import com.google.gson.Gson;
```

Comment:
Is `Gson` a plugin? My Android Studio's intellisence can't find it (but it still builds).

Reply:
Yes, I included it as a dependency in Gradle